README.md

```diff
@@ -544,6 +537,39 @@ they will be passed directly as `read_csv` arguments. For example:
         encoding: latin-1
 ```
 
+#### map_remove_forbidden_characters
```

Comment:
The name is quite long and will be quite difficult to remember for the end user. I am wondering whether we should use a shorter name. Even just `replace` seems to describe well enough what the transformer is doing.